stock_delivery_result_import/wizards/stock_delivery_result_import.py

```diff
@@ -72,16 +72,24 @@ def import_stock_delivery_result(self):
                 )
         if not import_log.error_ids:
             for picking, vals in pick_dict.items():
-                carrier_info = vals["carrier_info"]
+                if len(carrier_info) != 0:
+                    carrier_info_val = vals["carrier_info"]
+                else:
+                    carrier_info_val = False
```

Comment:
@TomohiroAbe-Quartile Not getting what you are trying to do here.  Can you first describe what issues you observe from debugging?